packages/common/src/search/types/IHubSearchResult.ts

```diff
@@ -0,0 +1,105 @@
+import { IHubGeography } from "../..";
+import { AccessLevel, HubFamily, IHubEntityBase } from "../../core";
+
+/**
+ * Standardized light-weight search result structure, applicable to all
+ * types of search results - users, groups, content, events etc
+ */
+export interface IHubSearchResult extends IHubEntityBase {
```

Comment:
Do we need a derived type or can we have EntityBase even if some of the attributes are null?

Reply:
Type-wise, we can use various [utility types](https://www.typescriptlang.org/docs/handbook/utility-types.html) if we need to narrow or widen `IHubEntityBase` at this `extends`. That said, I think all the `IHubEntityBase` props should have values for anything we can fetch

Reply:
More simply stated - do we need both `IHubSearchResults` and `IHubEntityBase`?

Reply:
At the very least, I think the `.meta` hash is something specifically for the `IHubSearchResult` that we don't want in the `IHubProject` etc. The entity specific interfaces should have _all the props_ already, so no need to have the .meta hash bag-of-holding